landing-page/scripts/tests/test_resolve_hints_staleness.py

```
@@ -0,0 +1,185 @@
+                    "centerline_page": {"x_start": 0, "x_end": 10, "y_values": [5] * 11}}]
+
+    cursor = FakeCursor(
+        annotations_row=(current_annotation_id, "2 0.5 0.3 0.9 0.02\n2 0.5 0.35 0.9 0.02"),
+        staffline_row_by_annotation_id={stale_annotation_id: (stale_jsomr,)},
+        text_alignment_row=None,
+    )
+    monkeypatch.setattr(tasks_encode, "get_db_conn", lambda: FakeConnection())
+    monkeypatch.setattr(tasks_encode, "release_db_conn", lambda con: None)
+    monkeypatch.setattr(FakeConnection, "cursor", lambda self: cursor, raising=False)
+
+    _text_alignment, yolo_stave_hints, stave_source = tasks_encode._resolve_hints(
```

Ruff (0.16.1)

[warning] Unpacked variable `yolo_stave_hints` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)